src/nvlink/worker_32bit.py

```
@@ -0,0 +1,377 @@
+
+        else:
+            result = {"success": False, "error": f"Unknown command: {command}"}
+
+    except Exception as e:
+        result = {"success": False, "error": str(e)}
+
+    finally:
+        # Always try to close
+        try:
+            worker.close()
+        except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)